src/http/ngx_http_upstream.c

```diff
@@ -1395,15 +1582,11 @@ ngx_http_upstream_connect(ngx_http_request_t *r, ngx_http_upstream_t *u)
 
     ngx_memzero(u->state, sizeof(ngx_http_upstream_state_t));
 
-    tp = ngx_timeofday();
-    u->state->response_sec = tp->sec;
-    u->state->response_msec = tp->msec;
-    u->state->header_sec = (time_t) NGX_ERROR;
+    u->state->response_time = ngx_current_msec;
+    u->state->connect_time = (ngx_msec_t) -1;
+    u->state->header_time = (ngx_msec_t) -1;
 
     rc = ngx_event_connect_peer(&u->peer);
-    if (rc == NGX_YIELD) {
-        return;
-    }
```

Comment:
delete by mistake?

Reply:
Yea thx,  This PR fix it: https://github.com/alibaba/tengine/pull/1151/commits/10e0d3b19e0aa30f92f0a35ce4d0da90319d7bf7